go/cmd/dolt/commands/diff.go

```diff
@@ -132,6 +139,7 @@ func (cmd DiffCmd) ArgParser() *argparser.ArgParser {
 	ap.SupportsInt(limitParam, "", "record_count", "limits to the first N diffs.")
 	ap.SupportsFlag(CachedFlag, "c", "Show only the unstaged data changes.")
 	ap.SupportsFlag(SkinnyFlag, "sk", "Shows only primary key columns and any columns with data changes.")
+	ap.SupportsFlag(MergeBase, "", "Uses merge base of {{.LessThan}}from_commit{{.GreaterThan}} and {{.LessThan}}to_commit{{.GreaterThan}} (or HEAD if not supplied) as {{.LessThan}}from_commit{{.GreaterThan}}")
```

Comment:
(minor) Might consider using "first commit" and "second commit" instead of "from_commit" and "to_commit", since I don't think we document them with those names in the rest of the docs. Alternatively, I actually like "from_commit" and "to_commit" better (seems easier for users to understand) if you wanted to switch to using those more widely.